diff --git a/dist/util/fs.js b/dist/util/fs.js
index 7b42b1c00753bf824d8caf55bd6ceed217eff0a2..a32f115b363642dda249a9a50d7fe7020192e5b6 100644
--- a/dist/util/fs.js
+++ b/dist/util/fs.js
@@ -1,6 +1,6 @@
 import { existsSync, readdirSync, realpathSync, statSync } from 'node:fs';
 import { readFile } from 'node:fs/promises';
-import { parse as parseYAMLContents } from 'yaml';
+import { load as parseYAMLContents } from 'yaml';
 import { parse as parseTOML } from 'smol-toml';
 import stripJsonComments from 'strip-json-comments';
 import { LoaderError } from './errors.js';
